modules/apps/questions/questions-web/src/main/java/com/liferay/questions/web/internal/portlet/QuestionsPortlet.java

```diff
@@ -177,11 +186,53 @@ private String _getTagSelectorURL(
 		}
 	}
 
+	private boolean _isTrustedUser(RenderRequest renderRequest) {
+		ThemeDisplay themeDisplay = (ThemeDisplay)renderRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+
+		try {
+			MBModerationGroupConfiguration mbModerationGroupConfiguration =
```

Comment:
Why can you use the configuration in the other module?

Reply:
As it was not an -api module, I wasn't sure if I was able (it was recommended) to export the configuration from the same module. 

Another option that came to my mind is to create a message-board-moderation-api, but as this was just for one class it seemed overkilling.

That is why I move it to message-board-api as it already exists, but you are completely right that we are coupling two modules together.

I will create a public package (now configuration is in an internal package) and export it from the message-board-moderation module, and use it from there.

Reply:
It is not possible to export the package from the existing module. I will create a message-boards-moderation-api to export it from there